website/docs/advanced/builder.md

```diff
@@ -6,7 +6,7 @@ sidebar_label: Configure MEV Builder
 
 import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';
 
-<HeaderBadgesWidget commaDelimitedContributors="James" lastVerifiedDateString="May 18th, 2023" lastVerifiedVersionString="v4.0.2" />
+<HeaderBadgesWidget commaDelimitedContributors="James" lastVerifiedDateString="May 18th, 2023"/>
```

Comment:
Should we change this last date string? So that it's apparent that it's been updated

Reply:
i think we should just remove this actually for now i'll do that